provisioning/roles/webpack/templates/webpack.config.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
         loader: 'babel-loader',
         options: {
           presets: [
-            ['env', {
+            ['@babel/preset-env', {
```

Comment:
In case user would later add a rule that also depends on Babel, for example a `vue-loader`, it would be better to move the babel config to an external and new `babel.config.js`. The `browserslist` import at the top could then be removed since `preset-env` now use browserslist automatically.